### ISSUE TITLE
husky init not working

### ISSUE THREAD

**Issue by sinbino** [typicode/husky#1334]
I am using docker node:20-slim image (debian 12.4, node 20.11.0, pnpm 8.14.3).
I get an error when I follow the [get started](https://typicode.github.io/husky/get-started.html) procedure.
The version of husky is 9.0.1

```
pnpm add --save-dev husky
pnpm exec husky init

node:internal/fs/utils:699
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
          ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined
    at Object.openSync (node:fs:579:10)
    at Object.readFileSync (node:fs:457:35)
    at file:///home/node/app/node_modules/.pnpm/husky@9.0.1/node_modules/husky/bin.js:9:24
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by recallwei** [comment]
Same to me, It seems `pnpm` not working, I use `npx` instead.

**Comment by marcinincreo** [comment]
`husky.sh` file that init creates is empty when using `npx`

**Comment by rozbo** [comment]
+1

**Comment by dmitriy-kudelko** [comment]
Same here

**Comment by artursudnik** [comment]
+1

**Comment by denisrudov** [comment]
just fixed in 9.0.2 👍🏻

**Comment by typicode** [comment]
It should be good now
